proposals/csharp-9.0/records.md

```diff
@@ -9,13 +9,34 @@ The syntax for a record is as follows:
 ```antlr
 record_declaration
     : attributes? class_modifier* 'partial'? 'record' identifier type_parameter_list?
-      parameter_list? record_base? type_parameter_constraints_clause* record_body
+      record_parameters? record_base? type_parameter_constraints_clause* record_body
     ;
 
 record_base
-    : ':' class_type argument_list?
+    : ':' class_type record_base_arguments?
     | ':' interface_type_list
-    | ':' class_type argument_list? ',' interface_type_list
+    | ':' class_type record_base_arguments? ',' interface_type_list
```

Comment:
Can `record_base` be simplified to the following? 
`| ':' class_type record_base_arguments? (',' interface_type_list)?`

Reply:
Yes! making that change.